styles.css

```diff
@@ -61,6 +64,57 @@ body {
   transform: scale(1.2);
 }
 
```

Comment:
- [ ] [OPTIONAL] The navigation links are not showing on the desktop version. I highly suggest you attend to this issue. 🥸️

![image](https://user-images.githubusercontent.com/17243325/232565587-e5138b29-0526-48e9-aa18-3677f242383b.png)